package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "chromedriver",
-  "version": "2.46.0",
+  "version": "73.0.3683.20",
```

Comment:
This would disrespect semver. Just keep it as `73.0.0`, because sometimes we need to rev the package for minor changes, and we need to be able to have the patch number available.

Reply:
Done